pkgs/applications/networking/cluster/rke2/update-script.sh

```diff
@@ -1,49 +1,41 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl git gnugrep gnused yq-go nix-prefetch
+#!nix-shell -i bash -p curl git gnugrep gnused yq-go nix-prefetch go
```

Comment:
Where is `go` being used in the script? I must have missed it.

Reply:
It's needed by the script from the rancher repository.